platform/spec/services/create_connection_and_syncs_service_spec.rb

```
@@ -0,0 +1,61 @@
+    let(:connection_id) { 123 }
+    let(:connection_service) { instance_double(Connections::CreateService) }
+    let(:syncs_service) { instance_double(Syncs::CreateService) }
+
+    before do
+      allow(Connections::CreateService).to receive(:new).with(connector.id).and_return(connection_service)
+      allow(connection_service).to receive(:call).and_return(connection_id)
+      allow(Syncs::CreateService).to receive(:new).with(connection_id).and_return(syncs_service)
+      allow(syncs_service).to receive(:call)
+    end
+
+    it 'creates a connection and syncs within a transaction' do
```

rubocop

[convention] Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

(Style/StringLiterals)